src/reducers/PatientListResultsReducer.js

```diff
@@ -0,0 +1,11 @@
+export default function(state = {patientEntries: [], meta:{}}, action)  {
+  switch (action.type) {
+    case 'LOAD_PATIENTS_FULFILLED':
+      const patientEntries = action.payload.data.entry.map((e) => e.resource);
+      const { total, link } = action.payload.data;
+      const meta = { total,link };
```

Comment:
As I explained on slack (recreating here) the meta is in fact the total number of patients in the selected filters. The patients array is the number of currently loaded patients.